resources/db/migration/V2__Add_image_size.sql

```diff
@@ -1,4 +1,4 @@
 -- store image size on upload
--- to not touch files when someone requests stats
+-- do not touch files when someone requests stats
```

Comment:
Please revert changes in this file.
One should not edit existing migrations, they are checked with checksum on app start.
If we merge this and deploy, the application will throw an exception and reject to start.